### PR TITLE
Scroll mouse wheel not work when View is smaller than content

### PR DESCRIPTION
**Pull request by syluna**
Separate check :
- width for scroll horizontal 
- height for scroll vertical

### CODE REVIEW

File: api/src/main/java/io/github/eckig/grapheditor/window/PanningWindow.java [hunk 1]
```diff
@@ -163,7 +153,10 @@ private boolean canNotPan()
     {
```
Comment (eckig) [inline api/src/main/java/io/github/eckig/grapheditor/window/PanningWindow.java:153]: Maybe rename to `canNotPanX()` to keep it consistent?

**Review by eckig — CHANGES_REQUESTED** [review]
Please adjust the formatting (braces on new line).

File: api/src/main/java/io/github/eckig/grapheditor/window/PanningWindow.java (hunk 1 above)
Comment (syluna) [reply api/src/main/java/io/github/eckig/grapheditor/window/PanningWindow.java:153]: I have commit asking change. I add document too on canPanY().

File: api/src/main/java/io/github/eckig/grapheditor/window/PanningWindow.java [hunk 2]
```diff
@@ -140,30 +140,29 @@ public void setEditorProperties(final GraphEditorProperties pEditorProperties)
...
-
-        final double newX = checkContentX(x);
-        final double newY = checkContentY(y);
-        if (newX != getContentX() || newY != getContentY())
-        {
-            contentX.set(newX);
-            contentY.set(newY);
-        }
+        panToX(x);
+		panToY(y)
```
Comment (eckig) [inline api/src/main/java/io/github/eckig/grapheditor/window/PanningWindow.java:144]: Maybe a `;`

**Review by eckig — CHANGES_REQUESTED** [review]
First of all thanks for looking into this.

Since this does not compile, I just have to ask: Did you try this? :)

File: api/src/main/java/io/github/eckig/grapheditor/window/PanningWindow.java (hunk 2 above)
Comment (syluna) [reply api/src/main/java/io/github/eckig/grapheditor/window/PanningWindow.java:144]: Arf no I can't test directly my env is set for my works so I can't change this. I just copy/past my change in web editor to apply the change.

Sorry....

**Review by syluna — COMMENTED** [review]
Fix semi colon

**Review by eckig — APPROVED** [review]

**Pull request merged**